crt0stack/src/builder.rs

```diff
@@ -477,7 +478,7 @@ mod tests {
             builder.push(&Entry::Random([0u8; 16])).unwrap();
 
             let handle = builder.done().unwrap();
-            let alignment = (handle.start_ptr() as *const _ as usize) % align_of::<Stack>();
+            let alignment = (&*handle as *const _ as usize) % align_of::<Stack>();
```

Comment:
I like to call `.deref()` directly to show, that it has a special meaning.

Reply:
While a nice idea, I think that goes against Rust idiom.